mds/models.py

```diff
@@ -165,11 +162,15 @@ def __str__(self):
 
     @property
     def latest_event(self):
-        if hasattr(self, "_latest_event"):
+        if hasattr(self, "_latest_events"):
             # don't do a query in this case, the telemetry was prefetched.
-            return self._latest_event[0] if self._latest_event else None
-        device = Device.objects.filter(pk=self.pk).with_latest_event().get()
-        return device.latest_event
+            return self._latest_events[0] if self._latest_events else None
+        latestEvent = (
```

Comment:
This is a list of latest event**s** :wink: 

And this is not the Python coding style to name variables.

Reply:
What do you mean ? I don't quite follow :'(

Reply:
`latestEvent` => `latest_events`